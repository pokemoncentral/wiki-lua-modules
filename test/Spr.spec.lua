-- "Test cases" for Spr

local spr = require("Spr")

-- Random thing
print(spr.spr({ args = { "479L", "Platino", "shiny back", "30px" } }))

-- Old gen, without gender
print(spr.Spr({ args = { "123", "Rubino", "male back shiny" } }))

-- Female only
print(spr.Spr({ args = { "478", "xy", "male" } }))

-- Current gen
print(spr.Spr({ args = { "133", "current", "male" } }))
