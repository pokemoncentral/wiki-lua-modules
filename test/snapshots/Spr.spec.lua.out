[[File:Sprptdsh479L.png|30px]]
[[File:Sprrzdsh123.png]]
[[File:Sprxyf478.gif|150px]]
[[File:133.png]]
